src/middlewared/middlewared/plugins/jail.py

```diff
@@ -146,15 +146,15 @@ class Config:
         process_pool = True
 
     OFFICIAL_REPOSITORIES = {
-        'https://github.com/freenas/iocage-ix-plugins.git'
+        'ixSystems': 'https://github.com/freenas/iocage-ix-plugins.git'
     }
```

Comment:
Sounds like this should be a list, because what if we want to add more metadata to each repository later on?

Reply:
Or a dict with another dict as values

Reply:
This seems most reasonable to me,
```
{'ix': {'git_repository': 'uri'}}
```
Thoughts ? This would mean that the friendly name would be unique and act as a key, else we can have a list and dicts inside

Reply:
The key can be an enum, e.g.

{"IXSYSTEMS": {..."name": "iXsystems", "git_repository": "..."...}, "COMMUNITY": {...}}

Reply:
Perfect, i'll update accordingly